src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/ServiceBusEntityParam.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.ServiceBus;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    internal class ServiceBusEntityParam
```

Comment:
Recommend renaming this something like ServiceBusEntityInfo

Reply:
we already class ServiceBusEntity defined. Renamed to ServiceBusEntityInfo.